.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+# We could use an image with pyenv & tox, but it's just as much work as doing a matrix build.
+# Sorry tox.ini...
+version: 2
+
+shared: &shared
+    steps:
+      - checkout
+      # Define config in CircleCi's project settings → Environment Variables
+      - run: echo \{\"app_id\":\"$FB_APP_ID\",\"app_secret\":\"$FB_APP_SECRET\",\"act_id\":\"act_$FB_ACCOUNT_ID\",\"page_id\":\"$FB_PAGE_ID\",\"access_token\":\"$FB_ACCESS_TOKEN\",\"business\":\"$FB_BUSINESS_ID\",\"sec_business\":\"$FB_BUSINESS_ID_BIS\",\"sec_act_id\":\"act_$FB_ACCOUNT_ID_BIS\"\} > config.json
```

Comment:
Just to confirm, but those won't be visible to public users? …

Reply:
As long as we don't open up accesses (SSH into builds) and "Pass secrets to builds from forked pull requests" remains turned off on CircleCi (for some reason it was enabled...), I'm not worried.
In any case, our CCi builds are not public at all currently (F/OSS mode).